spec/menu_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "menu.rb"
+
+describe Menu do
+
+  let(:menu) { Menu.new }
```

Comment:
A small thing, as you're testing the menu object here, it could be worth using the following instead of let:

` subject(:menu) { described_class.new }`

Reply:
thanks! been a bit scared of subjects, lets, doubles and the like! the always give the worst error msgs when you put them in the wrong place
Ill be sure to experiment in my next project!